notebooks/byod.py

```diff
@@ -98,30 +99,51 @@ def upload_columns(table: str, columns: Dict[str, List[Any]]):
             tsv_data += os.linesep + "\t".join([f"{i}", *[columns[h][i] for h in column_headers]])
         upload_data_table(tsv_data)
 
+    def parse_cram_crai(filename: str):
```

Comment:
Please remove `rsplit()` from this function as it produces edge case errors and explicitly only remove `.cram` or `.crai` or `.cram.crai`.